server/controllers/configurationsController/configurationsController.test.js

```diff
@@ -104,7 +106,23 @@ describe('ConfigurationsController', () => {
     describe('When successful', () => {
       let appDb
       let user
-
+      const [activeConfig, draftConfig, activeConfig2, legacyConfiguration] = [
+        createConfiguration({ _id: new ObjectId() }),
+        createConfiguration({ _id: new ObjectId(), status: 0 }),
+        createConfiguration({
+          _id: new ObjectId(),
+          readers: ['owl', 'eagle'],
+          owner: 'eagle',
+        }),
+        {
```

Comment:
why isn't this one using the fixture function?

Reply:
The fixture function adds the status property to the configuration, and this one is replicating a legacy configuration where the status property is non existent. This better represents the different documents we are bound to come in contact with.